superset/config.py

```diff
@@ -184,7 +184,10 @@ def _try_json_readsha(filepath: str, length: int) -> str | None:
 SECRET_KEY = os.environ.get("SUPERSET_SECRET_KEY") or CHANGE_ME_SECRET_KEY
 
 # The SQLAlchemy connection string.
-SQLALCHEMY_DATABASE_URI = "sqlite:///" + os.path.join(DATA_DIR, "superset.db")
+SQLALCHEMY_DATABASE_URI = (
+    f"""sqlite:///{os.path.join(DATA_DIR, "superset.db")}?check_same_thread=false"""
+)
```

Comment:
Can we add a note about this in the docs? If you search for `sqlite://` under `/docs` you can find a few occurrences where this could be worth mentioning.

Reply:
Added the extra property in a couple of examples in the docs and added an explanation in one place which seemed relevant as well.